pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java

```diff
@@ -187,7 +187,7 @@
 
     final IbftStateMachine ibftStateMachine = new IbftStateMachine(blockCreatorFactory);
     final IbftProcessor ibftProcessor =
-        new IbftProcessor(ibftEventQueue, ibftConfig.getRequestTimeoutMillis(), ibftStateMachine);
+        new IbftProcessor(ibftEventQueue, ibftConfig.getRequestTimeoutSeconds(), ibftStateMachine);
```

Comment:
Far side of this call is still referring to it's parameter as millis

Reply:
Done.